src/config-site/rzansel61.cmake

```diff
@@ -18,7 +18,7 @@ VISIT_OPTION_DEFAULT(VISIT_C_COMPILER /usr/tce/packages/gcc/gcc-7.3.1/bin/gcc TY
 VISIT_OPTION_DEFAULT(VISIT_CXX_COMPILER /usr/tce/packages/gcc/gcc-7.3.1/bin/g++ TYPE FILEPATH)
 VISIT_OPTION_DEFAULT(VISIT_FORTRAN_COMPILER no TYPE FILEPATH)
 VISIT_OPTION_DEFAULT(VISIT_C_FLAGS " -fPIC -fvisibility=hidden" TYPE STRING)
-VISIT_OPTION_DEFAULT(VISIT_CXX_FLAGS " -fPIC -fvisibility=hidden" TYPE STRING)
+VISIT_OPTION_DEFAULT(VISIT_CXX_FLAGS "-std=c++11  -fPIC -fvisibility=hidden" TYPE STRING)
```

Comment:
I don't think we need to set the standard in these flags.
In our CheckMinimumCompilerVersion.cmake file, we tell cmake which standard we want:
set(CMAKE_CXX_STANDARD 11)
set(CMAKE_CXX_STANDARD_REQUIRED True)

This should be sufficient, did you find it not to be the case? (Perhaps the 'True' setting should be changed to 'ON')

Reply:
I'll enter a ticket to look into this in the future.